src/test/java/org/jabref/logic/l10n/LocalizationConsistencyTest.java

```diff
@@ -183,6 +183,28 @@ public void localizationParameterMustIncludeAString() throws IOException {
         }
     }
 
+    @Test
+    public void localisationTestForInvalidStrings() throws IOException {
+        for (String bundle : Arrays.asList("JabRef", "Menu")) {
+            for (String lang : Languages.LANGUAGES.values()) {
+                String propertyFilePath = String.format("/l10n/%s_%s.properties", bundle, lang);
+
+                // read in
+                Properties textKeys = LocalizationParser
+                        .getProperties(propertyFilePath);
+                //parse object "textKeys" to find any spaces
+                for (Map.Entry<Object, Object> entry : textKeys.entrySet()) {
+                    String expectedKeyEqualsKey = String.format("%s=%s", entry.getKey(), entry.getKey());
+                    String actualKeyEqualsValue = String.format("%s=%s", entry.getKey(),
+                            entry.getValue().toString().replace(" ", ""));
+
+
+                    assertEquals("Found an invalid character in the "+ lang + "localization of" + bundle + ": The " + entry.getKey().toString() + ":" + expectedKeyEqualsKey + " contains a space!",expectedKeyEqualsKey, actualKeyEqualsValue);
```

Comment:
This looks fine, however make sure to include some spaces and quotes to make the sentence more readable
`Found an invalid character in the inlocalization ofJabRef: The pairs_processed:pairs_processed=pairs_processed contains a space! `
=>
`Found an invalid character in the 'in' localization of 'JabRef': The 'pairs_processed:pairs_processed=pairs_processed' contains a space! `

Also the error should be apparent from the message. The String above does not have a space.

Reply:
I give you push access. I'm correcting my code.

Reply:
I think it's ok now. But I didn't success to test ! It seems I don't know how to do.